src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationResultEnrichmentService.kt

```diff
@@ -88,6 +92,24 @@ class CalculationResultEnrichmentService(
     return hints.filterNotNull()
   }
 
+  private fun showSDS40Hints(calculationBreakdown: CalculationBreakdown): Boolean {
+    if (!featureToggles.sdsEarlyReleaseHints) return false
+
+    if (calculationBreakdown.concurrentSentences.any {
+        !it.isSDSPlus && (it.sentenceCalculationType.isSdsCalcType() && it.sentencedAt.isAfter(SDS_CUT_OFF_DATE))
+      }
+    ) {
+      return false
+    }
```

Comment:
Shame we don't have the sentence identification track here, would be a bit of a fudge getting it so agree what you've done is consistent 👍🏻

Reply:
I missed this scenario. The description mentions excluding SDS40+, which I may have taken too literally. I'll examine the ticket again.